eagleproject/core/blockchain/harvest/transaction_history.py

```diff
@@ -1127,7 +1128,24 @@ def send_report_email(summary, report, prev_report, report_type):
             'email': subscriber.email,
             'conf_num': subscriber.conf_num
         }))
-        result = e.execute([subscriber.email])
+        e.execute([subscriber.email])
+
+
+def send_report_email_core(summary, report, prev_report, report_type):
+    core = settings.CORE_TEAM_EMAIL
+    e = Email(summary, "test", render_to_string('analytics_report_email.html', {
```

Comment:
"test"??

Reply:
I never addressed this, this is plain_text version of the email ([see Django docs](https://docs.djangoproject.com/en/4.1/topics/email/#sending-alternative-content-types)) I would assume not many are still using email clients that are not able to render a html version of the email. 

We probably won't want to create double context for each email. Let me make a commit that strips HTML characters out of the rich content email and set that as plain text version

Reply:
done here: https://github.com/OriginProtocol/ousd-analytics/pull/298/commits/0797200bc9e0601218924220ea197905e4201a46